### ISSUE TITLE
[de.hafas.android.vvv] Does not display maps anymore

### ISSUE THREAD

**Issue by billie80** [microg/GmsCore#2100]
**Affected app**
Name: cleVVVer mobil
Package id: de.hafas.android.vvv

**Describe the bug**
At some point maps stopped to display in the App. Unfortunately I can not pinpoint it to a specific App, microG or OS release. It supports two different maps Google Maps and basemap.at (which uses a [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/deed.de) license).

**To Reproduce**
1. Open App and Go to the Map (Karte) Tab
2. No map is displayed

**Expected behavior**
The App is showing the map as expected.

**System**
Android Version: 10
Custom ROM: ShiftOS-L 3.10 (20231025)

**microG**
microG Core version: 0.2.27.223616
microG Self-Check results: All ticked

**Additional context**
On the same device with the same OS but with GMS instead of microG the map is displayed. I already though about contacting the App developers but I fear the obvious answer is "We do not support Android without GMS". If you can not fix this issue maybe you can give tips on how to approach the developers, like giving hints which changes might fix the issue. This way chances are higher for the issue getting resolved upstream.

**Comment by Wojtaz0w** [comment]
Try updating microG

**Comment by Wojtaz0w** [comment]
@billie80 it works with microG 0.3.0
![Screenshot_20231128-071600_cleVVVer mobil.png](https://github.com/microg/GmsCore/assets/104674625/f7f62ad3-819c-41fb-973a-6649d39a11a5)

**Comment by Wojtaz0w** [comment]
For future reference, when reporting bugs always test on latest version

**Comment by billie80** [comment]
Thanks for letting me know!
Usually I test the newest version, but with the new location stack I decided to stay with the stable version until this new versions are officially recommended.

**Comment by Wojtaz0w** [comment]
Please close this issue, reminding you cause you seem to have forgotten about it

**Comment by billie80** [comment]
Or I close it if a stable version of microG which I can test is available. If maintainers disagree they can close it.

**Comment by Wojtaz0w** [comment]
YOU ARE 3 STABLE RELEASES BEHIND UPSTREAM

**Comment by billie80** [comment]
Firstly, I don't like being shouted at for no reason, so please refrain from doing so.
Secondly, I don't know what authority you have here to demand that bugs be closed. I don't see that you are part of this project. So again, if a team member closes the bug, that's fine with me. But I would like to leave it open.
Thirdly, just because it's possible to install any version doesn't mean it's stable. The microG team doesn't distinguish on Github whether a release is stable or not but I think there are good reasons why they don't recommend the versions from 0.2.27 in their F-Droid repository which I use.

**Comment by Wojtaz0w** [comment]
yeah, I overeacted. Nonetheless, **official** F-Droid repo of microG reccomends version 0.3.0.![Screenshot_20231213-181640_Neo Store.png](https://github.com/microg/GmsCore/assets/104674625/60ddb57b-3138-4b77-993f-81d011a66bce)

**Comment by billie80** [comment]
Strangely the official F-Droid Client says otherwise. I just synced the repository again to make sure.

![F-Droid_microG](https://github.com/microg/GmsCore/assets/9938306/d916a565-5c0a-4dea-97ca-c18776ff2b47)

However the last update to the repo it was on October 28. Maybe I am missing something.

**Comment by billie80** [comment]
Works now again with newer versions of microg. Was not using microg for a while so couldn't test.
